ods-core/jenkins/ocp-config/bc.env.sample

```diff
@@ -8,3 +8,6 @@ JENKINS_SLAVE_BASE_IMAGE=Dockerfile.centos7
 
 # Openshift application base dns - used for grabbing the root ca and adding into the slave
 APP_DNS=192.168.99.100.nip.io
+
+# configuration which slave base to use
+JENKINS_SLAVE_BASE_FROM_IMAGE=openshift/jenkins-slave-base-centos7
```

Comment:
I know this is a super-nitpick, but from http://www.opendevstack.org/doc/development-guide:

> End all files with a newline character. See the POSIX definition of a line.

This helps to keep diff to a minimum.

Reply:
okay, will eof